src/Yoti/Util/Profile/AttributeConverter.php

```diff
@@ -65,18 +69,50 @@ private static function convertValueBasedOnContentType(ProtobufAttribute $protob
                 // Convert JSON string to an array
                 $value = json_decode($value, true);
                 if (json_last_error()) {
-                    throw new AttributeException("Error converting attr {$attrName} to a JSON Object");
+                    throw new AttributeException("Error converting attr to a JSON Object");
                 }
                 break;
 
             case self::CONTENT_TYPE_DATE:
                 $value = self::convertTimestampToDate($value);
                 break;
+
+            case self::CONTENT_TYPE_MULTI_VALUE:
+                $value = self::convertMultiValue($value);
+                break;
         }
 
         return $value;
```

Comment:
Could we also add `case self::CONTENT_TYPE_STRING:`, and also for `CONTENT_TYPE_UNDEFINED` here, (and add them to the constants at the top). That way it is clearer that they're not just cases that we haven't considered, and it's clear what we are doing with them.

Reply:
I've reorganised the switch statement to include `CONTENT_TYPE_STRING`/`CONTENT_TYPE_UNDEFINED` and return values.